docs/INSTALLATION.md

```diff
@@ -1,13 +1,15 @@
 ## Dart
-* [Install Dart](https://www.dartlang.org/install)
-* Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* One option is to follow the [instructions for installation on the Dart website](https://www.dartlang.org/install).
 
-## Dart Plugin for your IDE or Editor
-* If you are using an IDE or an editor, [install and configure the Dart plugin](https://www.dartlang.org/tools).
+### Manual install instruction
```

Comment:
If there is a 'Manual installation instructions' there should be an 'Automatic installation instructions' too.

Reply:
Those instructions are on the dart-lang website. Since we link to it, we don't need to duplicate the steps here otherwise, they may change in the future depending on the platform.

Reply:
No, I was talking about the section titles, so if there's a manual installation section there should be an automatic installation section too.

Reply:
Consider the following structure :

# Installation   


### Automatic   


###  Manual